.github/workflows/update-nextcloud-ocp.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branches: ["master", "stable25", "stable24"]
+        branches: ["main", "master", "stable25", "stable24"]
```

Comment:
That will not work, as `composer require --dev nextcloud/ocp:dev-main` does not work.
You need to further modify the workflow to use another install branch there